lib/room.js

```diff
@@ -396,15 +396,21 @@ function connectParticipant(room, participantSignaling) {
     'trackSubscriptionFailed',
     'trackUnpublished',
     'trackUnsubscribed'
-  ].map(event => {
+  ].map(eventOrEventPair => {
```

Comment:
@markandrus This can be reverted back to the simpler re-emit logic, since "trackAdded" and "trackRemoved" are no longer emitted.